### ISSUE TITLE
Juzi's legs 

### ISSUE THREAD

**Issue by jmalkovich18** [itsamewayu/OLFallenDoll#2020]
Just suggestion,
 Maybe possible, say, to put Juzi's legs in  the shop and make available to buy them. I believe there is many players who want to see Juzi with full body. 
  To be honest i don't really understand why this lady look like that in many experiences. 

 With respect. Thanks

**Comment by 67879017** [comment]
支持！

**Comment by batterystapled** [comment]
Agree, There are some really good experiments that would be better with legs present, eg 649, 651, 652, 653,
